torch/distributions/binomial.py

```diff
@@ -98,8 +103,11 @@ def log_prob(self, value):
                 self.total_count * torch.log1p((self.logits + 2 * max_val).exp()))
 
     def enumerate_support(self):
-        values = self._new((self.total_count,))
-        torch.arange(self.total_count, out=values.data)
+        total_count = max(int(self.total_count.max().item()), 1)
+        if not self.total_count.min() == total_count:
+            raise NotImplementedError("Inhomogeneous total count not supported by method.")
+        values = self._new(total_count,)
+        torch.arange(total_count, out=values)
```

Comment:
I think this was just a bug. This should have been `torch.arange(1 + total_count)` since the actual values allowed are `[0, ..., total_count]` inclusive.

Reply:
Thanks, yes, let me push a fix for this.